neural_compressor/torch/algorithms/weight_only/modules.py

```diff
@@ -47,6 +48,170 @@ def forward(self, X):
 
 
 class WeightOnlyLinear(torch.nn.Module):
+    def __init__(
```

Comment:
Is the `WeightOnlyLinear` device-agnostic? If so, please clean the device-related functions.

Reply:
Fixed in https://github.com/intel/neural-compressor/pull/1877/commits/56c864f58cee53be0a79e816e5686bbe1fffbce1. `WeightOnlyLinear` base class is device-agnostic now.